src/commenttarget.js

```diff
@@ -0,0 +1,63 @@
+const SEPARATOR = '/';
+
+async function parseCommentTarget(opts = {}) {
+  const { commitSha: commit, pr, target, drv } = opts;
+
+  let commentTarget = target;
+  // Handle legacy comment target flags.
+  if (commit) {
+    drv.warn(
+      'cml: the --commitSha flag will be deprecated, please use --target="commit/<sha>"'
+    );
+    commentTarget = `commit${SEPARATOR}${commit}`;
+  }
+  if (pr) {
+    drv.warn('cml: the --pr flag will be deprecated, please use --target="pr"');
```

Comment:
```suggestion
    drv.warn('Deprecation warning: use --target=pr instead of --pr');
```

Reply:
Fixed in https://github.com/iterative/cml/pull/1228/commits/dcb37b323fc75872563c163f260bdaf534e76747